.travis.yml

```diff
@@ -1,10 +1,16 @@
+sudo: false
+
 language: ruby
+
+bundler_args: ''
```

Comment:
hmm it does a `bundle install  --path=${BUNDLE_PATH:-vendor/bundle}` so that might work out just fine ...